playbook-website/config/webpack/environment.js

```diff
@@ -13,6 +13,30 @@ environment.loaders.insert('javascript', {
   include: path.resolve(__dirname, '../../app'),
 })
 
+environment.loaders.prepend('svgr', {
+  test: /\.(svg)$/,
+  use: {
+    loader: '@svgr/webpack',
+  },
+  include: [
+    path.resolve(__dirname, '../../../node_modules/@powerhome/playbook-icons/icons')
+  ],
+})
+
+// Don't let file entry stomp on SVGs
+environment.loaders.get('file').test = /(.jpg|.jpeg|.png|.gif|.tiff|.ico|.eot|.otf|.ttf|.woff|.woff2)$/i
+environment.loaders.get('file').include = path.resolve(__dirname, '../../app')
+
+environment.loaders.append('image', {
+  test: /\.(svg)$/,
+  use: {
+    loader: 'file-loader',
+  },
+  include: [
+    path.resolve(__dirname, '../../app')
+  ],
+})
+
```

Comment:
`file-loader` from webpack was stepping on the custom `@svgr/webpack` loader since it was just lumped into the regex by default. This removes that `|.svg` entry and allows the other to grab the SVGs we want to grab. Lastly, we then specifically tell webpack to load remaining SVGs using `file-loader`. This was necessary because of the ancient version of webpack we are using.